libevm/VM.cpp

```diff
@@ -214,19 +214,24 @@ void VM::interpretCases()
 		
 		CASE(CREATE)
 		{
+			if (m_schedule->haveStaticCall && m_ext->staticCall)
```

Comment:
Do we really need a check for `m_schedule->haveStaticCall` here and in other places?
I think it's redundant, it can't be a static call if don't have it 😄